src/powershell-win-core/Microsoft.PowerShell_profile.ps1

```diff
@@ -0,0 +1,10 @@
+# Default profile for all users for PowerShell host
+
+if ($IsWindows)
+{
+	# Add Windows PowerShell PSModulePath to make it easier to discover potentially compatible PowerShell modules
+	# If a Windows PowerShell module works or not, please provide feedback at https://github.com/PowerShell/PowerShell/issues/4062
+
+	Write-Warning "Appended Windows PowerShell PSModulePath"
+	$env:psmodulepath += ";${env:userprofile}\Documents\WindowsPowerShell\Modules;${env:programfiles}\WindowsPowerShell\Modules;${env:windir}\system32\WindowsPowerShell\v1.0\Modules\"

```

Comment:
Calling out to Windows PowerShell can incur a 150-250ms startup penalty.  Since this change isn't intended to be the final solution and just helps to get customer feedback, I think we are ok with it not being 100% accurate.  Advanced customers can update their PSModulePath as needed.